python/wpt/update.py

```diff
@@ -126,8 +128,7 @@ def run_update(**kwargs) -> bool:
 
 def create_parser(**_kwargs):
     parser = wptcommandline.create_parser_update()
```

Comment:
Should we also update the argument in the [WPT import script](https://github.com/servo/servo/blob/master/.github/workflows/scheduled-wpt-import.yml#L55)